KaranTeam/KaranTeam/Data/Entities/User.cs

```diff
@@ -12,6 +12,7 @@ public class User : IdentityUser
     {
         public bool IsAdmin { get; set; } = false;
         public string Name { get; set; }
+        public string Email { get; set; }
```

Comment:
```suggestion
```
ez nem kell, mert amiből leszármazik a user az tudja ezt, amúgy a Name-sem kell mert van beépített UserName, de azt majd utána javítom én a közös ágon

Reply:
A többi helyen a beépített UserName-et használod ami jó.